tempesta_fw/sock.c

```diff
@@ -114,14 +114,25 @@ ss_skb_entail(struct sock *sk, struct sk_buff *skb)
 	tcb->seq     = tcb->end_seq = tp->write_seq;
 	tcb->tcp_flags = TCPHDR_ACK;
 	tcb->sacked  = 0;
-	skb_header_release(skb);
+	__skb_header_release(skb);
```

Comment:
Comment for `__skb_header_release()` says "assuming skb is private to caller", so what if we forward a skb received from local web-server process (the process also takes reference to the skb)? Or do you handle the case by uncloning?..